charts/clearml/templates/apiserver-deployment.yaml

```diff
@@ -75,8 +75,15 @@ spec:
             value: {{ include "elasticsearch.servicename" . }}
           - name: CLEARML_ELASTIC_SERVICE_PORT
             value: "{{ include "elasticsearch.serviceport" . }}"
+          {{- if .Values.mongodb.enabled }}
           - name: CLEARML_MONGODB_SERVICE_CONNECTION_STRING
             value: {{ include "mongodb.connectionstring" . | quote }}
+          {{- else }}
+          - name: CLEARML__HOSTS__MONGO__BACKEND__HOST
+            value: {{ .Values.externalServices.mongodbConnectionStringBackend | quote }}
+          - name: CLEARML__HOSTS__MONGO__AUTH__HOST
+            value: {{ .Values.externalServices.mongodbConnectionStringAuth | quote }}
+          {{- end }}
```

Comment:
I see one issue with this:
it won't work with the simplest case, which is setting `mongodb.auth.enabled=true` & `mongodb.auth.rootPassword="password123"`.

Reply:
Using a custom values dependent mongodb chart can generate a lot of different combinations we can't support directly. In case you described, it should be better to simply disable mongodb chart and install a custom one before installing Clearml with mongodb disabled.
If you have a better PR to propose, I'm more than happy to drop this one and discuss on yours.